### ISSUE TITLE
AMA Intake | disable previously selected issues on Add issues page

### ISSUE THREAD

**Issue by suzchap** [department-of-veterans-affairs/caseflow#7012]
## Story
For any AMA forms, when a user needs to add issues, they will be walked through a multi-screen selection process to help them make the best choice. The landing page created in #6966 will be their starting point and a modal on top of it will walk the user through rated, non-rated, and an "unknown" option.

When a user adds a rated issue, the next time the add an issue, the add rated issues page will show that first rated issue but will prevent it's selection by disabling the radio button and display the text as gray, italic, and appended with "already selected for issue X". (note: making "X" match the correct issue number is out of scope for this issue and will be handled separately).

requires: #6981 

## Acceptance criteria
- When a user clicks "+Add issue" button on the "Add issue page" (#6966), it overlays a modal showing timely decisions and rating issues but any previously selected issues for that claim are not selectable.
- Previously selected issues are also gray and in italic and followed by "(already selected for issue X)

## Copy
- (already selected for issue X)

## Mocks
<img width="672" alt="screen shot 2018-09-17 at 3 59 26 pm" src="https://user-images.githubusercontent.com/11219417/45647253-6e66f480-ba93-11e8-95c0-5976c0f31801.png">

## How to Validate
- Doesn't require a specific test user
- Form select > search id > review page: any answers > on "Add issue page, click "+Add issue" > select a rated issue > click "Add this issue" button > on "Add issue page, click "+Add issue" > you should see your first issue grayed out, italic, and appended with extra language from AC. 
- TBD: does this need to be tested with all 3 AMA forms or will one suffice?

**Comment by shanear** [comment]
<img width="495" alt="screen shot 2018-10-05 at 2 45 20 pm" src="https://user-images.githubusercontent.com/1596259/46554144-570a7280-c8ad-11e8-98ac-0fdad32b9def.png">
looking good
